levelpacker/levelpacker.py

```diff
@@ -210,15 +211,30 @@ def handle_tileset(ts: minidom.Node, objs: GameObjects, builder: flatbuffers.Bui
         FlatBuffGenerated.Tileinfo.TileinfoAddGid(builder, gid)
         objs.tileinfo.append(FlatBuffGenerated.Tileinfo.TileinfoEnd(builder))
 
+def handle_tile_layer(layer: minidom.Node, builder: flatbuffers.Builder) -> str:
```

Comment:
Nit: does this function really return a `str`? From looking at line 231 it seems that it returns an offset, so it should be an `int` instead.